Doc/library/pdb.rst

```diff
@@ -245,6 +245,9 @@ access further features, you have to do this yourself:
    .. versionadded:: 3.14
       Added the *mode* argument.
 
+   .. versionchanged:: 3.14
+      *skip* will be ignored if inline breakpoints like :func:`breakpoint` or :func:`set_trace` are used.
```

Comment:
"if ... are used" could be misunderstood to mean that skip will *always* be ignored if those are *ever* used.

Reply:
Will changing it to "when ... are used" better? Or we should just use the longer version from the news entry?

Reply:
I think "when .. are used" has the same problem, as does the longer version from the news entry.

Reply:
```
Inline breakpoints like :func:`breakpoint` or :func:`pdb.set_trace` will always stop the program at calling frame, ignoring the ``skip`` pattern (if any).
```

Is this better?

Reply:
Yes.

Reply:
Then I'll use this for all 3 places :)